src/components/StoryList/Header.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { css } from 'react-emotion'
+import * as MainSiteStyles from '../../globals/mainsiteGlobalStyles'
+
+interface HeaderProps {
+  title: string
+}
+
+export default function Header(props: HeaderProps) {
+  return (
+    <div
+      className={css`
+        background-color: #000;
+        padding: 0.125rem 0rem 0.125rem 0.5rem;
+      `}
+    >
+      <h2
+        className={css`
+          color: #fff;
+          font-family: ${MainSiteStyles.topBarFont}, sans-serif;
+          font-size: 1.1875rem;
```

Comment:
Why are some of these values very specific?

Reply:
It's more of me being nitpicky with how it looks than anything 😅 I could change these values to "less specific" ones if you want

Reply:
Would it maybe be better to use pixels if that's the case? For font size maybe not so much, but for things like padding, margin, etc.

Reply:
I can do that,  but I'm kinda worried about how the component would scale if I used px for the padding and margins tho

Reply:
Yeah - let's use pixels, but add them with names like `CardInnerPadding` etc. so we can have a consistent look across main site components